themes/digital.gov/layouts/shortcodes/checklist.html

```diff
@@ -0,0 +1,34 @@
+{{/* 
+  
+  Sample code for creating a checklist with a list of checkbox items
+ 
+  1. Use the checklist shortcode when you want to have a printable checklist
+  2. Provide a "header" and the checklist will set a gray bar beneath an H2
+  3. Nest {{< checkbox >}} shortcodes within the {{< checklist >}} to create a list of checkbox items
+  4. Use {{< checkbox-sublist-item sublist-item="sublist bullet point">}} will create a bullet point sublist item
+  5. Follow the same nesting structure as below, don't indent {{< checkbox-sublist-item >}} in the {{< checkbox >}} shortcode, else it will break
+
+  {{< checklist header="Preparing your presentation" >}}
+
+    {{< checkbox item="Item number 1 goes here" >}}
+    {{< checkbox-sublist-item sublist-item="detail one" >}}
+    {{< checkbox-sublist-item sublist-item="detail two" >}} 
+    {{< /checkbox >}}
+
+    {{< checkbox item="Item number 2 goes here, with supporting lines that wrap to the next if this is long enough" >}}  {{< /checkbox >}}
+    {{< checkbox item="Item number 3 goes here" >}}{{< /checkbox >}}
+  
+  {{< /checklist >}} 
+  
+*/}}
+
+
+{{- if .Inner -}}
+<ul class="dg-checklist">
+  {{- if (.Get "header") -}}
+    <h2 class="dg-checklist__header">{{ .Get "header" }}</h2>
```

Comment:
`<h2>` as a child of `<ul>` is invalid. Axe devtools also shows 3 errors:

<img width="1213" alt="image" src="https://user-images.githubusercontent.com/3385219/225661747-46ab1b18-565f-4b23-9c12-728b89141cbd.png">

Reply:
Great catch, I've fixed the `h2` issue. The second one will be addressed by #5935